packages/legacy-workbench/src/js/angular/plugin-registry.js

```diff
@@ -52,37 +51,66 @@ window.PluginRegistry = (function() {
         return typeof variable !== 'undefined';
     }
 
-    function registerPlugin(extensionPoint, pluginDefinition) {
-        if (pluginDefinition.disabled) {
-            return;
-        }
+    function getExtensionPointIdFieldName(extensionPointName) {
+      if ('guide.step' === extensionPointName) {
```

Comment:
umm what? The plugin registry should be unaware of the concrete extension points

Reply:
Maybe I didn’t understand you, but we decided to make the unique identifier abstract and dependent on the extension point. In the context of guides, it is `guideBlockName`. I’ll think of another way to implement it.